chrome/app/record_replay_main.cc

```diff
@@ -22,6 +22,29 @@
 
 #endif // !BUILDFLAG(IS_WIN)
 
+static void ReportFailure(const char* format, ...) {
+  {
+    va_list args;
+    va_start(args, format);
+    vfprintf(stderr, format, args);
+    fprintf(stderr, "\n");
+    va_end(args);
+  }
+#if BUILDFLAG(IS_WIN)
+  // Additionally write the message to a new file. Capturing the output written to
+  // stderr by browser subprocesses on windows is surprisingly difficult.
+  FILE* f = fopen("record_replay_load_recorder_error.txt", "w");
```

Comment:
Opening the file in "w" mode will truncate it, no?  So this won't be a log so much as just the last error message.  I think you want "w+" here.